db/db.go

```diff
@@ -132,170 +134,136 @@ var dbops = map[string][]string{
 	},
 }
 
-// dblogger is the logger for the database subsystem.
+// dbLogger is the logger for the database subsystem.
 var (
-	dblogger = util.NewLogger("system", "db")
+	dbLogger = util.NewLogger("system", "db")
 )
 
 // Dber is an interface that returns a reference to the underlying *sql.DB
 type Dber interface {
 	Db() *sql.DB
 }
 
-// TimeoutDber composes Dber and GetTimeouter to return timeout duration for dbops
+// TimeoutDber composes Dber and GetTimeouter to return timeout duration for dbOps
 type TimeoutDber interface {
 	Dber
 	util.GetTimeouter
 }
 
-type getdboper interface {
-	getdbop(string) string
+type queryProvider interface {
+	getQuery(dbOp) string
 }
 
-type dbOper struct {
-	t int
+type mapProvider struct {
+	dbType int
 }
 
-func newPostgressDbOper() *dbOper {
-	return &dbOper{
-		t: postgres,
+func newPostgressDbOper() *mapProvider {
```

Comment:
You should probably update the function name to match that it's a mapProvider and not a dbOper